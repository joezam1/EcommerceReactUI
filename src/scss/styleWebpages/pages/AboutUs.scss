*{
  box-sizing:border-box;
}

.about-us-container{
    
    height: 100%;
    overflow-y: auto;
    position: fixed;
    text-align:center;
    width: 100%;
    z-index: 5500;
  }
  
  .about-us-page{
    background-color:white;
    border: 1px solid #ddd;
    border:1px solid red;
    border-radius:5px;
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
    display:inline-block;
    max-height: 1200px;
    padding:20px;
    position: relative;
    text-align:center;
    top:50px;
    max-width: 700px;
  }

  