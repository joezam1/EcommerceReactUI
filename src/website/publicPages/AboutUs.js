import React from 'react';
import NavigationFooter from '../home/footer/NavigationFooter';
import ButtonGoHome from '../common/ButtonGoHome.js';

export default function AboutUs(){

  
    return(<div className="about-us">
                <div className="about-us-background"></div>
                <div className="about-us-container">
                    <div className="about-us-page">
                        <ButtonGoHome/>
                        <div><NavigationFooter/></div> 
                        <br/><br/><br/>
                        <div className="about-us-page-content">
                            <h2>ABOUT US</h2>

                            <p>We are an Australian based online jewellery store.
                            Our website houses an impeccable blend of iconic designs.
                            Excellent customer service is the secret to our success
                            and you can have confidence that the highest standards of service
                            and integrity will be maintained in our dealings with you.</p>

                            <p>From this website you can see our gorgeous variety, create a wish list,
                            and shop for numerous styles creating a unique personal statement for you.
                            All our orders will ship within 24 -48 hours (subject to availability).
                            </p>
                            <p>
                            If you have a question, please donâ€™t hesitate to ask via the Contact Us page.
                            </p>
                            <p>
                            We hope you enjoy our wonderful products.
                            </p>
                        </div>
                    </div>
                </div>
            </div>)
      
}