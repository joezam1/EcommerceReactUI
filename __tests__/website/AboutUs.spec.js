import React from 'react';
import {render, cleanup, waitFor, fireEvent} from '@testing-library/react';

import AboutUs from '../../src/website/publicPages/AboutUs.js';

xdescribe('file AboutUs', function(){
    afterEach(cleanup);
    test('create a snapshot', function(){
        let container = render(<AboutUs/>);
        expect(container.asFragment(<AboutUs/>)).toMatchSnapshot();
    })
})
