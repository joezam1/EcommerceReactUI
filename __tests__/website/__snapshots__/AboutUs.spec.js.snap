// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file AboutUs create a snapshot 1`] = `
<DocumentFragment>
  <div
    class="about-us"
  >
    <div
      class="about-us-background"
    />
    <div
      class="about-us-container"
    >
      <div
        class="about-us-page"
      >
        <div
          class="button-go-home"
        >
          <span
            class="button-go-home-text"
            data-testid="button-go-home-id"
          >
            ×
          </span>
        </div>
        <div>
          <div
            class="navigation-footer"
          >
            <div
              class="footer-elements-ul"
            >
              <div
                class="footer-element-li"
                data-testid="home"
                id="home"
              >
                Home
              </div>
              <div
                class="footer-element-li"
                data-testid="about-us"
                id="about-us"
              >
                About Us
              </div>
              <div
                class="footer-element-li"
                id="terms-and-conditions"
              >
                Terms & Conditions
              </div>
              <div
                class="footer-element-li"
                id="shippings-and-returns"
              >
                Shippings & Returns
              </div>
              <div
                class="footer-element-li"
                id="privacy-policy"
              >
                Privacy Policy
              </div>
              <div
                class="footer-element-li"
                id="security"
              >
                Security
              </div>
            </div>
          </div>
        </div>
        <br />
        <br />
        <br />
        <div
          class="about-us-page-content"
        >
          <h2>
            ABOUT US
          </h2>
          <p>
            We are an Australian based online jewellery store. Our website houses an impeccable blend of iconic designs. Excellent customer service is the secret to our success and you can have confidence that the highest standards of service and integrity will be maintained in our dealings with you.
          </p>
          <p>
            From this website you can see our gorgeous variety, create a wish list, and shop for numerous styles creating a unique personal statement for you. All our orders will ship within 24 -48 hours (subject to availability).
          </p>
          <p>
            If you have a question, please don’t hesitate to ask via the Contact Us page.
          </p>
          <p>
            We hope you enjoy our wonderful products.
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
